R/shortcuts.R

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Province of British Columbia
+# Copyright 2023 Province of British Columbia
```

Comment:
What about dropping the date? More folks seem to be doing this.

Reply:
I'll have a look at the Apache template (or other new bcgov code), happy to drop if that is the new way.